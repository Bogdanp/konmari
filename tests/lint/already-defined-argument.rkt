#lang racket/base

(define (f g g)
  g)
