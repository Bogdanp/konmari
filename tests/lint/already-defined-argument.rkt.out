/already-defined-argument.rkt:3:14:error:identifier 'g' is already defined
/already-defined-argument.rkt:3:10:warning:identifier 'f' is never used
